### DISCUSSION TITLE
Exemple of an element of type Lsp.Types.Diagnostic.t

### DISCUSSION THREAD

**Discussion by SoilihBenSoilih** [ocaml/ocaml-lsp#394] [General]
Hello, 

I'm currently developping an LSP server in ocaml, I have some issues with the diagnostics. Can someone show me how an element of Lsp.Types.Diagnostic.t type look like? Perhaps I will figure out my issue after that.

Thank you a lot !!!

**Comment by Gbury** [comment]
I don't know what you mean by "show me how an element of Lsp.Types.Diagnostic.t type look like", but if if helps, here is some code that creates diagnostics for the lsp server in one of my projects : https://github.com/Gbury/dolmen/blob/master/src/lsp/diagnostic.ml

**Reply by SoilihBenSoilih** [reply]
I was meaning like if an element of type diagnostic it's a record who look like v={name:"Soilih";age:24}.
Thank you for the response !!!
